solver.rb

```diff
@@ -0,0 +1,5 @@
+class Solver
+  def reverse(str)
+    str = str.reverse
+  end
+end
```

Comment:
- [ ] By the look of things you wrote the method before writing the tests. As the name implies TDD, you need to write the tests first before the method implementation. 

See screenshot of your commit history:
![a1](https://user-images.githubusercontent.com/39523312/157693019-7afa0ade-434a-4493-8edb-d7779134fddd.png)